mlir/utils/jenkins/Jenkinsfile

```diff
@@ -107,12 +107,64 @@ pipeline {
                      description: 'Run the performance testing stage')
     }
     stages {
+        stage("Set System Property") {
+            steps {
+                script {
+                     System.setProperty("org.jenkinsci.plugins.durabletask.BourneShellScript.HEARTBEAT_CHECK_INTERVAL", "86400");
```

Comment:
What's this do? Why do we need it?

Reply:
This increases the interval for checking dead processes. I've seen the following issue a couple of times: 

http://ml-ci.amd.com:21096/blue/organizations/jenkins/MLIR%2Frefactor-Jenkinsfile/detail/refactor-Jenkinsfile/17/pipeline/

The problem might be that the job thought the node was no longer alive. Hope this script will help.

Reply:
I don't think we should be `setProperty`ing in Jenkinsfiles, and that that's instead a systemwide config change (same is the content-security-policy stuff needed to make the perf reports pretty). So, we shouldn't have this here and instead @ Omkar (not sure what his github is) to get this applied to the cluster. In the meantime, this sort of thing should be done with the Jenkins system console.

Reply:
@krzysz00  it's better to have within the mlir pipeline that way it would only be applied to mlir jobs, making changes to systemwide jenkins would impact all the other jobs

Reply:
@okakarpa I could've sworn that system.setProperty() is inherently global...